prime_server/http_protocol.hpp

```diff
@@ -190,4 +194,19 @@ struct http_response_t : public http_entity_t {
 
 using http_server_t = server_t<http_request_t, http_request_info_t>;
 
+constexpr uint8_t ALL_VERBS_MASK = OPTIONS | GET | HEAD | POST | PUT | DELETE | TRACE | CONNECT;
+
+shortcircuiter_t<http_request_t> make_shortcircuiter(const std::string& health_check_path  = "/health_check",
+                                                     const uint8_t& verb_mask = ALL_VERBS_MASK,
+                                                     const std::string& allowed_origins = "*",
+                                                     const std::string& allowed_headers = "*",
+                                                     const int max_age = 86400);
+
+std::unordered_set<std::string> parse_string_list(const std::string& parameter_str);
```

Comment:
move this to the source file there is no need to expose it in the header

Reply:
Done